src/api/tasks/TaskCollaborators.js

```diff
@@ -15,6 +15,8 @@ import {
 } from '../../constants';
 import type { ElementsErrorCallback } from '../../common/types/api';
 import type { BoxItem } from '../../common/types/core';
+import Xhr from '../utils/Xhr';
```

Comment:
Can we use the underlying `post` method on the API classes to avoid interacting with the XHR class directly?